helpers/skills/notebooks-cve-resolution/SKILL.md

```
@@ -0,0 +1,200 @@
+
+```bash
+make refresh-lock-files
+```
+
+Check if CVE is resolved in lockfiles. If NOT resolved, add constraint.
+
+### Add CVE constraint
+
+Edit `dependencies/cve-constraints.txt`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)